vcnc-server/vcnc-sampler/app.js

```diff
@@ -87,31 +84,51 @@ function requestHandler(request, response) {
   response.end();
 }
 
-const server = http.createServer(requestHandler);
-server.setTimeout(5000, () => {
-});
+// Create server
+//
+let server;
+function startServer(h, p) {
+  server = http.createServer(requestHandler);
+  server.setTimeout(5000, () => {
+  });
```

Comment:
You are right. server.timeout doesn't require callback. I used it probably it was require when I wrote original code. I remove it